src/renderer/viz/expressions/linear.js

```diff
@@ -30,7 +31,7 @@ import TZDate from '../../../utils/time/TZDate';
 * @api
 */
 export default class Linear extends BaseExpression {
-    constructor (input, min, max, range) {
+    constructor (input, min, max, range, samples = DEFAULT_SAMPLES) {
```

Comment:
What is the use case for introducing here this optional param?

Reply:
Same as here: https://github.com/CartoDB/carto-vl/pull/1323#discussion_r275337504